openidm-zip/src/main/resources/db/h2/scripts/openidm.sql

```diff
@@ -0,0 +1,482 @@
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS idx_managedobjects_object ON openidm.managedobjects (objecttypes_id,objectid);
+-- Note that the next two indices apply only to role objects, as only role objects have a condition or temporalConstraints
+-- CREATE INDEX IF NOT EXISTS idx_json_managedobjects_roleCondition ON openidm.managedobjects
+--    ( json_extract_path_text(fullobject, 'condition') );
+-- CREATE INDEX IF NOT EXISTS idx_json_managedobjects_roleTemporalConstraints ON openidm.managedobjects
+--    ( json_extract_path_text(fullobject, 'temporalConstraints') );
+--H2 doesn't recognize json_extract_path
+
+-- -----------------------------------------------------
+-- Table openidm.managedobjectproperties
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.managedobjectproperties (
+  managedobjects_id BIGINT NOT NULL,
+  propkey VARCHAR(255) NOT NULL,
+  proptype VARCHAR(32) DEFAULT NULL,
+  propvalue TEXT,
+  CONSTRAINT fk_managedobjectproperties_managedobjects FOREIGN KEY (managedobjects_id) REFERENCES openidm.managedobjects (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+
+CREATE INDEX IF NOT EXISTS fk_managedobjectproperties_managedobjects ON openidm.managedobjectproperties (managedobjects_id);
+-- CREATE INDEX IF NOT EXISTS idx_managedobjectproperties_prop ON openidm.managedobjectproperties (propkey,propvalue);
+
+-- -----------------------------------------------------
+-- Table openidm.configobjects
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.configobjects (
+  id BIGSERIAL NOT NULL,
+  objecttypes_id BIGINT NOT NULL,
+  objectid VARCHAR(255) NOT NULL,
+  rev VARCHAR(38) NOT NULL,
+  fullobject JSON,
+  PRIMARY KEY (id),
+  CONSTRAINT fk_configobjects_objecttypes FOREIGN KEY (objecttypes_id) REFERENCES openidm.objecttypes (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+
+CREATE UNIQUE INDEX IF NOT EXISTS idx_configobjects_object ON openidm.configobjects (objecttypes_id,objectid);
+CREATE INDEX IF NOT EXISTS fk_configobjects_objecttypes ON openidm.configobjects (objecttypes_id);
+
+-- -----------------------------------------------------
+-- Table openidm.configobjectproperties
+-- -----------------------------------------------------
+
+CREATE TABLE IF NOT EXISTS openidm.configobjectproperties (
+  configobjects_id BIGINT NOT NULL,
+  propkey VARCHAR(255) NOT NULL,
+  proptype VARCHAR(255) DEFAULT NULL,
+  propvalue TEXT,
+  CONSTRAINT fk_configobjectproperties_configobjects FOREIGN KEY (configobjects_id) REFERENCES openidm.configobjects (id) ON DELETE CASCADE ON UPDATE NO ACTION
+);
+
+CREATE INDEX IF NOT EXISTS fk_configobjectproperties_configobjects ON openidm.configobjectproperties (configobjects_id);
+-- CREATE INDEX IF NOT EXISTS idx_configobjectproperties_prop ON openidm.configobjectproperties (propkey,propvalue);
```

Comment:
Same.

Reply:
Index removed. `json_extract_path*` is postgres-specific. 
This index doesn't seem to be crucial, because current non-postgres scripts don't create it.